include/materials/norsand.tcc

```diff
@@ -406,9 +404,9 @@ void mpm::NorSand<Tdim>::compute_plastic_tensor(const Vector6d& stress,
     dR_dj3 *= std::pow(j2, -1.5);
     // Update derivative of theta in terms of R, check for sqrt of zero
     if (abs(1 - R * R) < 1.0E-6) {
-      dtheta_dR = 1.0 / 3.0 / sqrt(1.0E-6);
+      dtheta_dR = -1.0 / 3.0 / sqrt(1.0E-6);
     } else {
-      dtheta_dR = 1.0 / 3.0 / sqrt(1 - R * R);
+      dtheta_dR = -1.0 / 3.0 / sqrt(1 - R * R);
```

Comment:
Could you please use parenthesis notation, serial division is likely to be confusing